private/bufpkg/bufcheck/client.go

```diff
@@ -491,27 +491,25 @@ func ignoreLocation(
 }
 
 // checkCommentLineForCheckIgnore checks that the comment line starts with the configured
-// comment ignore prefix, and the rest of the string is the ruleID of the check.
+// comment ignore prefix, a space and the ruleID of the check.
 //
-// We currently do not support multiple rules per comment ignore:
+// All of the following comments are valid, ignoring SERVICE_PASCAL_CASE and this rule only:
 //
-//	Invalid:
-//		// buf:lint:ignore SERVICE_SUFFIX, SERVICE_PASCAL_CASE
+//	// buf:lint:ignore SERVICE_PASCAL_CASE, SERVICE_SUFFIX
```

Comment:
Is this saying that you can do comma-separated ignores? I don't think that's in scope

Reply:
Updated comment in [f516a9c](https://github.com/bufbuild/buf/pull/3295/commits/f516a9c597e110b619aed297cde8d96e723b3f3a) (it also said `ignoring SERVICE_PASCAL_CASE and this rule only` on top)